src/Microsoft.AspNetCore.SignalR.Client.Core/Internal/ConnectionLogScope.cs

```diff
@@ -30,7 +30,7 @@ public string ConnectionId
         {
             get
             {
-                if (Count == 1 && index == 0)
+                if (index == 0)
```

Comment:
I think this is fine. The only case where you actually get `null` here is if the value is `null`ed out in between when the consumer calls `.Count` and when the consumer dereferences the indexer.